magicctapipe/scripts/lst1_magic/lst1_magic_event_coincidence.py

```diff
@@ -138,7 +142,32 @@ def telescope_positions(config):
     return TEL_POSITIONS
 
 
-def event_coincidence(input_file_lst, input_dir_magic, output_dir, config):
+def func1(X, a, b):
+    """
+    The linear function to fit the MAGIC timestamp and difference from LST.
+
+    Parameters
+    ----------
+    X : numpy array
+        MAGIC timestamp
+    a : float
+        Intersept
+    b : float
+        Slope
+
+    Returns
+    -------
+    float
+        The time offset value obtained from the MAGIC timestamp
+    """
+
+    Y = a + b * X
+    return Y
+
+
+def event_coincidence(
+    input_file_lst, input_dir_magic, output_dir, input_dir_toff, config
```

Comment:
you could set input_dir_toff by default to None since it is only used in this special circumstances